client/pages/Login/Login.jsx

```diff
@@ -0,0 +1,34 @@
+// React lib import
+import React from 'react';
+
+// styles
+import './Login.scss';
+
+/**
+ * Represents Login component
+ *
+ * @class Login
+ * @extends {React.Component}
```

Comment:
I don't think this is correct right? Quickly checkup the difference between presentational and container components and see how it could help us here